src/detector.jl

```diff
@@ -144,18 +146,57 @@ function ssd_config(meta::PropDict, env::Environment)
     cone_r_top = cylinder_radius - meta.geometry.taper.top.outer.height_in_mm * tan(meta.geometry.taper.top.outer.angle_in_deg / 180. * π)
     cone_height = meta.geometry.taper.top.outer.height_in_mm
 
-    charge_density_model = Dict(
-        "name" => "cylindrical",
-        "r" => Dict(
-            "init" => 0,
-            "gradient" => 0
-        ),
-        "z" => Dict(
-            "init" => -1e7,
-            "gradient" => 1e5
+    ## grid 
+    # kind of initialize so that it exists out of the loop?    
+    grid = Dict()
+
+    if ssd_sim.coord == "cylindrical"
+        grid = Dict(
+            "axes" => Dict(
+                "r" => Dict("to" => cylinder_radius*1.2, "boundaries" => "inf"), # leave some margin
+                "phi" => Dict("from" => 0, "to" => ssd_sim.comp == "3D" ? 360 : 0, "boundaries" => "periodic"),
+                "z" => Dict(
+                    "from" => -10,
```

Comment:
I would suggest to use `-0.2*cylinder_height` here

Reply:
Done!